functions/src/express_configs/express_secure.ts

```diff
@@ -38,7 +62,7 @@ function errorHandler(error: Error, request: Request, response: Response, next:
 app.use(errorHandler);
 
 // Automatically send uncaught exception errors to Sentry
-process.on("uncaughtException", (err) => Sentry.captureException(err));
+process.on("uncaughtException", (error) => Sentry.captureException(error));
```

Comment:
You may want to rename the lambda variable (`error` to `err` or `e`) so there isn't any issue with name scope. There's already an `error` variable defined in the function header.